setup.py

```diff
@@ -991,6 +1031,29 @@ def determine_sdl2():
     else:
         print('AVFoundation cannot be used, OSX >= 10.7 is required')
 
+if c_options["use_angle_gl_backend"]:
+
+    # kivy.graphics.egl_backend.egl_angle is always compiled,
+    # but it only acts as a proxy to the real implementation.
+
+    if platform in ("darwin", "ios"):
```

Comment:
Is it not already implied by `c_options['use_angle_gl_backend'] = platform in ['darwin', 'ios']` from the same file?

Reply:
ATM yes, but the plan is to introduce the `ANGLE` backend even for platforms different than `darwin` and `ios`.
So, here we check if we can use the `angle_gl_backend` and then build the correct implementation (e.g. Metal, Android, ...)

That acts more like a placeholder for the future, but I'm open to removing it.

Reply:
Ah yeah that makes sense let's keep it